tests/serialize/json_test.py

```diff
@@ -98,55 +98,55 @@ def node_sort_key(node: dict[str, Any]) -> str:
         'id': '1',
         'label': 'to_string',
         'kind': 'function',
-        'function': 'json_test.to_string',
+        'function': 'tests.serialize.json_test.to_string',
```

Comment:
No, I didn't expect this. It is caused by `--import-mode=importlib` in pytest that was added in this PR.

Reply:
... but is this change fine?

Reply:
I'd say so. This is a particular detail of how pytest works. In a normal Python process, it should always use the full module path.